src/solvers/prop/prop.h

```diff
@@ -21,10 +21,11 @@ Author: Daniel Kroening, kroening@kroening.com
 
 /*! \brief TO_BE_DOCUMENTED
 */
-class propt:public messaget
+class propt
 {
 public:
   propt() { }
```

Comment:
Hmm, don't you need to consume a `message_handlert` here?

Reply:
I wasn't there before so adding one instead of constructing a new one could potentially change the behavior.

Reply:
Yes, it makes things safer, so I'd happily take that change.